components/payments/settings/payment-webhooks.tsx

```diff
@@ -7,5 +21,210 @@ interface PaymentsWebhooksProps {
 export const PaymentsWebhooks: React.FC<PaymentsWebhooksProps> = ({
   accountId,
 }) => {
-  return <Flex flexDir="column" gap={8}></Flex>;
+  const { data: webhookApiKey, isLoading: isLoadingSecretKey } =
+    usePaymentsWebhooksSecretKeyByAccountId(accountId);
+  const {
+    data: webhooks,
+    isLoading: isGetLoading,
+    isFetched,
+  } = usePaymentsWebhooksByAccountId(accountId);
+  const { mutate: updateWebhook, isLoading: isUpdateLoading } =
+    usePaymentsUpdateWebhook(accountId);
+  const { mutate: createWebhook, isLoading: isCreateLoading } =
+    usePaymentsCreateWebhook(accountId);
+
+  const toast = useToast();
+
+  const triggerAlert = (
+    status: "error" | "success",
+    header: string,
+    description: string,
+  ) => {
+    toast({
+      position: "bottom",
+      variant: "solid",
+      title: header,
+      description,
+      status,
+      duration: 9000,
+      isClosable: true,
+    });
+  };
+
+  const { productionWebhooks, testnetWebhooks } = useMemo(() => {
```

Comment:
this is a great improvement from before!

You can only return once if you have an optional param, this looks a bit cleaner to me but both work

`
  const { productionWebhooks, testnetWebhooks } = useMemo(
    () => ({
      productionWebhooks:
        webhooks?.filter((webhook) => webhook.isProduction) || [],
      testnetWebhooks:
        webhooks?.filter((webhook) => !webhook.isProduction) || [],
    }),
    [webhooks],
  );
`